recipe/meta.yaml

```diff
@@ -12,9 +12,10 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 4
+  number: 0
   skip: true  # [python_impl == 'pypy']
   skip: true  # [py>310]
+  skip: true  # [py<38]
```

Comment:
*That's* how you exclude Python versions being built?!?!? For real? That's hacky....

Reply:
I'll be honest, not 100% sure that's the "right" way, but given we exclude py>310 that way, I went with it (and it works)

Reply:
Fair!

Reply:
Huh, looks like it is the right way: https://conda-forge.org/docs/maintainer/adding_pkgs.html#build

Reply:
Ah, you found the docs that I couldn't find ;-). Alright, This Is The Way™.